### DISCUSSION TITLE
HFB vertical flow

### DISCUSSION THREAD

**Discussion by jonathanqv** [MODFLOW-ORG/modflow6#2072] [General]
Hi all! The documentation states that we cannot simulate vertical flow for HFBs, which means we cannot specify an HFB between layers, right? In MODFLOW-USG I see this option available. Is this an error in the documentation? are we able or not to do vertical flow with HFBs? If not, is there a specific reason why we cannot do this?

Thanks!

**Comment by christianlangevin** [comment]
Hi @jonathanqv, we have been slow to implement vertical flow barriers in MODFLOW 6 for a number of reasons.   Most importantly, there are conceptual differences between vertical flow barriers and horizontal flow barriers that potentially might affect a number of MODFLOW 6 features.  We would need to think all of that through before implementing.  With that said, it would be good to know if there is broad demand for this capability or if it is needed in just a few specialized cases.

**Reply by jonathanqv** [reply]
Thanks! I think my case is very specific. Looking forward to it in the future,
